lib/upgrade_format_task.rb

```diff
@@ -34,15 +34,21 @@ def upgrade(store, name, options = {})
 
     descriptions.each do |description|
       begin
+        hash = Manifest.load(description, store.manifest_path(description)).to_hash
         Machinery.logger.info "Upgrading description \"#{description}\""
-        Machinery::Ui.puts "Upgrading description \"#{description}\""
+        Machinery::Ui.print "Reading \"#{description}\" ... "
         migrated = Migration.migrate_description(store, description, force: options[:force])
-        migrations_done += 1 if migrated
+
+       if migrated
+          migrations_done += 1
+          Machinery::Ui.puts "successfully upgraded from version #{hash["meta"]["format_version"]} to #{SystemDescription::CURRENT_FORMAT_VERSION}."
```

Comment:
Line is too long. [148/100]<br>Prefer single-quoted strings inside interpolations.